godtools/App/Share/Data/TranslationsRepository/ManifestParser/TranslationManifestParser.swift

```diff
@@ -15,15 +15,20 @@ class TranslationManifestParser {
     private let parserConfig: ParserConfig
     private let resourcesFileCache: ResourcesSHA256FileCache
     
-    static func getManifestParser(type: TranslationManifestParserType, resourcesFileCache: ResourcesSHA256FileCache) -> TranslationManifestParser {
+    static func getManifestParser(type: TranslationManifestParserType, appConfig: AppConfig, resourcesFileCache: ResourcesSHA256FileCache) -> TranslationManifestParser {
         
         switch type {
-        case .related:
-            return ParseTranslationManifestForRelatedFiles(resourcesFileCache: resourcesFileCache)
+        
+        case .manifestAndRelatedFiles:
+            let parserConfig = ParserConfig().withParseRelated(enabled: false)
```

Comment:
I think you meant `enabled: true` here?

otherwise this is the same config as `.manifestOnly`

Reply:
hmm, maybe you did mean `enabled: false`. what are the use-cases that drive `.manifestAndRelatedFiles` and `.manifestOnly`? should they just be a single type?

Reply:
Hey @frett so parse related true here does that just attach the related files to the manifest?

I will also update this to set parsePages and parseTips to false since that parsing isn't needed when downloading and cacheing the manifest and related files to the file cache.

Reply:
taking a look at where you are using the 2 types, I think they can just be a single type of `.manifestOnly`.

Reply:
no matter the parser mode the `manifest.relatedFiles` property will always return a list of all related files, there is no parser mode to disable that behavior.

Reply:
the flags just indicate if the parser follows that list of related files to actually parse those files or not.